new-installer/targets/aws/iac/utils/utils.go

```diff
@@ -33,6 +33,7 @@ import (
 const (
 	DefaultTestRegion         = "us-west-2"
 	DefaultTestCustomerTag    = "test-customer"
+	DefaultTestRoute53Zone    = "iteptest.infra-host.com"
```

Comment:
This only works for our account.
Is there a way that community user can check this out and still make it work in their own aws account?